Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3917,7 +3921,7 @@ RefPtr<StyleValue> Parser::parse_transform_value(Vector<StyleComponentValueRule>
             } else if (value.is(Token::Type::Number)) {
                 auto number = parse_numeric_value(value);
                 values.append(number.release_nonnull());
-            } else {
+            } else if (!value.is(Token::Type::Comma) && !value.is(Token::Type::Whitespace)) {
```

Comment:
This logic isn't quite right - it allows things like `this(,,,,,,,,,,)`. CSS requires that we ignore any properties that we don't recognize or that don't fit the grammar, which means we need to catch weird cases like this and return nullptr.

Long term we probably want to make a JSON file of transform-functions and generate some code for checking their arguments, but for now: after successfully appending to `values`, look ahead and make sure that either we get a `Whitespace? Comma Whitespace?` sequence, or that we reach the last value without seeing a comma. That'll mean changing this loop to go by index.